src/nri/sampler_test.go

```diff
@@ -111,3 +122,80 @@ func TestECSLabelRename(t *testing.T) {
 		}
 	}
 }
+
+func TestExitedContainerTTL_Expired(t *testing.T) {
+	mocker := &mocker{}
+	mocker.On("ContainerList", mock.Anything, mock.Anything).Return([]types.Container{
+		{
+			ID:      "containerid",
+			Names:   []string{"Container 1"},
+			Image:   "my_image",
+			ImageID: "my_image_id",
+		},
+	}, nil)
+	mocker.On("ContainerInspect", mock.Anything, mock.Anything).Return(types.ContainerJSON{
+		ContainerJSONBase: &types.ContainerJSONBase{
+			State: &types.ContainerState{
+				Status:     "exited",
+				FinishedAt: time.Now().Add(-2 * time.Hour).Format(time.RFC3339Nano),
```

Comment:
I'd probably add some more margin here just to be in the safe side, like:
```suggestion
				FinishedAt: time.Now().Add(-2 * time.Hour - time.Minute).Format(time.RFC3339Nano),
```
We don't want a leap second to ruin our test :)

Reply:
Hahahahaha, it won't! I'm using 30 mins as TTL in the test. 😂

Reply:
> Hahahahaha, it won't! I'm using 30 mins as TTL in the test. 😂

That's cheating!! Hahha

Jokes aside it is fine to me in that case :)